examples/fungible-token/tests/workspaces.rs

```diff
@@ -18,11 +22,54 @@ async fn register_user(contract: &Contract, account_id: &AccountId) -> anyhow::R
     Ok(())
 }
 
-async fn init(
-    worker: &Worker<impl DevNetwork>,
+// TODO: in order to be able to use a once fixture from rstest (which uses std::sync::OnceLock<T>)
```

Comment:
What is expected TODO here? Make it async in the future? Is it worth it?

Reply:
TODO replaced by https://github.com/near/near-workspaces-rs/pull/386